manifests/repo.pp

```diff
@@ -8,18 +8,27 @@
 
   ensure_packages(['apt-transport-https', 'lsb-release', 'ca-certificates'], {'ensure' => 'present'})
 
-  if ($::operatingsystem == 'Ubuntu') {
+  if ($os::name == 'Ubuntu') {
```

Comment:
Why were these changes needed? Have they been tested on AWS as well as CCL?

Reply:
Nothing tested yet - but `operatingsystem` is a puppet legacy fact.

Need to test in both platforms still

Reply:
It might be better to put this change into a different change so you are not blocking the PHP fix with these changes needing to be tested